tracer/src/Datadog.Trace/RemoteConfigurationManagement/Protocol/RcmClientTracer.cs

```diff
@@ -35,6 +36,9 @@ public RcmClientTracer(string runtimeId, string tracerVersion, string service, s
         [JsonProperty("service")]
         public string Service { get; }
 
+        [JsonProperty("process_tags")]
+        public List<string> ProcessTags { get; }
```

Comment:
Is it correct that we send an empty list here when `tracerSettings.PropagateProcessTags == false`? Or should we omit the property entirely (e.g. by passing in null to the constructor)?